downstream/modules/platform/ref-controller-credential-openShift.adoc

```diff
@@ -0,0 +1,31 @@
+[id="ref-controller-credential-openShift"]
+
+= OpenShift or Kubernetes API Bearer Token
+
+Select this credential type to create instance groups that point to a Kubernetes or OpenShift container. 
+
+For more information, see link:https://docs.ansible.com/automation-controller/4.4/html/administration/containers_instance_groups.html#ag-ext-exe-env[Container and Instance Groups] in the Automation controller Administration Guide.
+
+//image:credentials-create-containers-credential.png[Credentials- create Containers credential]
+
+Container credentials require the following inputs:
+
+* *OpenShift or Kubernetes API Endpoint* (required): the endpoint used to connect to an OpenShift or Kubernetes container
+* *API Authentication Bearer Token* (required): The token used to authenticate the connection
+* Optional: *Verify SSL*: You can check this option to verify the server's SSL certificate is valid and trusted. 
+Environments that use internal or private _Certificate Authority_ (CA) must leave this option unchecked to disable verification.
+* *Certificate Authority Data*: include the `BEGIN CERTIFICATE` and `END CERTIFICATE` lines when pasting the certificate, if provided.
+
+A ContainerGroup is a type of InstanceGroup that has an associated Credential that enables connection to an OpenShift cluster. 
+To set up a container group, you must have the following:
+
+* A namespace you can launch into (every cluster has a “default” namespace, but you may want to use a specific namespace).
+* A service account that has the roles that enable it to launch and manage Pods in this namespace.
+* If you use {ExecEnvShort}s in a private registry, and have a Container Registry credential associated with them in {ControllerName}, the service account also requires the roles to get, create, and delete secrets in the namespace. 
+
```

Comment:
```suggestion
+
```
The bullet points are off 
Suggesting a "+", but you might want different formatting